src/main/scala/is/hail/expr/Type.scala

```diff
@@ -1313,19 +1308,12 @@ object TGenotype {
   
   def representationWithVCFAttributes(required: Boolean = false): TStruct = {
     val t = TStruct(IndexedSeq(
-      Field("GT", TCall(), 0,
-        Map("Number" -> "1", "Type" -> "String", "Description" -> "Genotype")),
-      Field("AD", TArray(TInt32()), 1,
-        Map("Number" -> "R", "Type" -> "Integer",
-          "Description" -> "Allelic depths for the ref and alt alleles in the order listed")),
-      Field("DP", TInt32(), 2,
-        Map("Number" -> "1", "Type" -> "Integer", "Description" -> "Read Depth")),
-      Field("GQ", TInt32(), 3,
-        Map("Number" -> "1", "Type" -> "Integer", "Description" -> "Genotype Quality")),
-      Field("PL", TArray(TInt32()), 4,
-        Map("Number" -> "G", "Type" -> "Integer",
-          "Description" -> "Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification"))))
-    
+      Field("GT", TCall(), 0),
+      Field("AD", TArray(TInt32()), 1),
+      Field("DP", TInt32(), 2),
+      Field("GQ", TInt32(), 3),
+      Field("PL", TArray(TInt32()), 4)))
+
```

Comment:
This should probably be written in the usual `TStruct` style now, as in `representation` above.

Moreover, is this even necessary anymore? Is it just the capitalization that's needed?